app/services/case_information_service.rb

```diff
@@ -7,4 +7,16 @@ def self.get_case_information(offender_ids)
       [case_info.nomis_offender_id, case_info]
     }.to_h
   end
+
+  def self.ldu_changed?(team_ids)
+    return false if team_ids.blank?
+
+    return false if team_ids.last.nil? # from eng/wales to scot/ni
+
+    return true if team_ids.first.nil? # from scot/ni to eng/wales
+
+    old_ldu = Team.find_by(id: team_ids.first).try(:local_divisional_unit)
+    new_ldu = Team.find_by(id: team_ids.last).try(:local_divisional_unit)
+    old_ldu != new_ldu
+  end
```

Comment:
This method feels very similar to some of the 'dity' methods which come with ActiveRecord models. (relevant docs: [here](https://api.rubyonrails.org/classes/ActiveModel/Dirty.html) and [here](https://api.rubyonrails.org/classes/ActiveRecord/AttributeMethods/Dirty.html))

It makes me think that this method would probably be more at home in the `CaseInformation` model, rather than in the `CaseInformationService` object.

I'm thinking that we'd be able to call something like `@case_info.was_ldu_changed?` after having called `@case_info.update`.

Reply:
+1